ProteoformSuiteGUI/ExperimentTheoreticalComparison.cs

```diff
@@ -87,6 +88,10 @@ public void ClearListsAndTables()
             Lollipop.proteoform_community.relations_in_peaks.RemoveAll(r => r.relation_type == ProteoformComparison.et || r.relation_type == ProteoformComparison.ed);
             Lollipop.proteoform_community.delta_mass_peaks.RemoveAll(k => k.relation_type == ProteoformComparison.et || k.relation_type == ProteoformComparison.ed);
 
+
+            foreach (var series in ct_ET_Histogram.Series) series.Points.Clear();
```

Comment:
Nice work clearing these charts.

Could you also add code to clear the datagridviews in ProteoformFamilies? Right now, it just clears the Lollipop list.